packages/core/utils/entityTypes.ts

```diff
@@ -56,9 +57,14 @@ export interface Token {
 	metadata?: AssetMetadata;
 }
 
+export interface Collection {
+	network: string;
+	metadata?: AssetMetadata;
+}
+
 export interface Collectible {
 	network: string;
-	collectionId?: string;
+	collectionId: string;
 	metadata?: AssetMetadata;
 }
```

Comment:
@ltminhthu here is type shape of collectible (aka nft) which includes metadata field as same as token type. In `send feature` we mostly use this field to render. It might be helpful to notice.